### ISSUE TITLE
Issues when uploading image files in Korean.

### ISSUE THREAD

**Issue by KimDaesap** [Requarks/wiki#65]
If upload a file name in Korean, the file name will not be reflected normally.
e.g) "이미지 1.png" -> "-1.png"

**Comment by NGPixel** [comment]
Non-alphanumerical characters are stripped at the moment. I'll try to add support for non-latin characters soon.

**Comment by KimDaesap** [comment]
Thanks!

**Comment by NGPixel** [comment]
Fixed for Beta 11 (fe313baf67cfd73d5eee041f96ec3b27be59c16a)

**Comment by KimDaesap** [comment]
This isuue not solved.

**Comment by choicky** [comment]
I did a test in my wiki. Images having a name containing or begining with Chinese Character is correctly uploaded and inserted.

FYI. [https://law.ipinfo.cn/仅用于测试的页面](https://law.ipinfo.cn/仅用于测试的页面 )

**Comment by KimDaesap** [comment]
I use wiki.js in Windows.
Maybe for that reason.
If possible, I will also test Linux and OSX.

**Comment by NGPixel** [comment]
I develop on Windows too and it works fine from the example you provided in your original post.

Do you have an example where it doesn't work?

**Comment by KimDaesap** [comment]
Resolved.

The previous wiki.js version was 1.0.0-beta10 and I created a separate folder and installed version 1.0.0-beta11 using 'npm install wiki.js@latest'.

D: / wiki (beta10)
D: / wiki-new (beta11)

I used 'npm start wiki' command in the folder where beta11 was installed, but beta10 was still running. I checked it using 'pm2 show wiki' command.

I've completely removed beta10 and reinstalled it.
